### PR TITLE
feat: expose update method

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
When changing textarea value directly it is not resizing itself, instead of mounting observers and make bigger library I believe that just exposing `update` method will help users controls sizing of autoresize. 
Useful especially in frameworks where model is often change directly.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -31,5 +31,8 @@ export const attach = (element: HTMLTextAreaElement) => {
     detach() {
       element.removeEventListener("input", inputHandler);
     },
+    update() {
+      inputHandler();
+    },
```
Comment (FRSgit) [inline src/index.ts:36]:
```suggestion
    update: inputHandler,
```

Comment (FRSgit) [reply src/index.ts:36]: But I would actually think about adding something like that directly to the attach function:
```ts
const elementPrototype = Object.getPrototypeOf(element);
const descriptor = Object.getOwnPropertyDescriptor(elementPrototype, 'value');
Object.defineProperty(element, 'value', {
    ...descriptor,
    set() {
        descriptor.set.apply(this, arguments);
        inputHandler();
    },
});
```

So as you said - observer in this library code. But IMO it's better to have it handled once here than having to do it separately in every library out there.

**Review by FRSgit — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (Szymon-dziewonski) [reply src/index.ts:36]: so maybe, both?

Comment (FRSgit) [reply src/index.ts]: Yea, having an update method exposed to the public is cool imo

**Review by FRSgit — APPROVED** [review]

**Pull request merged**